SPECS/python-sphinx_rtd_theme/python-sphinx_rtd_theme.spec

```diff
@@ -36,17 +36,16 @@ It's currently in development and includes some rtd variable checks that
 can be ignored if you're just trying to use it on your project outside
 of that site.
 
-%package -n python3-%{srcname}
-%{?python_provide:%python_provide python3-%{srcname}}
+%package -n python%{python3_pkgversion}-%{srcname}
```

Comment:
What does theputhon3_pkgversion macro resolve to?

Reply:
The number 3